src/components/Table/TableBody/TableBody.jsx

```diff
@@ -130,8 +131,7 @@ const TableBody = ({
   shouldExpandOnRowClick,
   shouldLazyRender,
   ordering,
-  wrapCellText,
-  truncateCellText,
```

Comment:
so we are removing 2 props and replacing with a single prop, isn't that a breaking change?

Reply:
See deprecation comment above